tests/test_routes.py

```diff
@@ -23,11 +24,11 @@ def test_create_fund(flask_test_client, _db, clear_test_data):
         "welsh_available": "false",
         "short_name": "NF5432",
         "funding_type": FundingType.COMPETITIVE.value,
+        "ggis_scheme_reference_number": "G1-SCH-0000092415",
```

Comment:
If this is the format dont we need more validation on the field ?

Reply:
its a string with 255 characters max length